recipes/unicorn/all/patches/2.0.0.-0001-cmake-msvc-support.patch

```diff
@@ -0,0 +1,9 @@
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -1416,7 +1416,7 @@ if(UNICORN_BUILD_TESTS)
+ endif()
+ 
+ 
+-if(UNICORN_INSTALL AND NOT MSVC)
++if(UNICORN_INSTALL)
+     include("GNUInstallDirs")
```

Comment:
Needs a newline at the end.
Also, the patch is incomplete. There need to be 7 lines.